cmd/fleet/serve.go

```diff
@@ -683,11 +679,18 @@ func startSchedules(
 	config config.FleetConfig,
 	license *fleet.LicenseInfo,
 	enrollHostLimiter fleet.EnrollHostLimiter,
+	failingPoliciesSet fleet.FailingPolicySet,
 	instanceID string,
 ) error {
+	appConfig, err := ds.AppConfig(ctx)
+	if err != nil {
+		return fmt.Errorf("getting app config: %w", err)
+	}
+
 	startCleanupsAndAggregationSchedule(ctx, instanceID, ds, logger, enrollHostLimiter)
 	startSendStatsSchedule(ctx, instanceID, ds, config, license, logger)
 	startVulnerabilitiesSchedule(ctx, instanceID, ds, logger, &config.Vulnerabilities, license)
+	startAutomationsSchedule(ctx, instanceID, ds, logger, 5*time.Minute, *appConfig, failingPoliciesSet)
```

Comment:
I see you changed another func to get AppConfig from ds, you have ds here, any reason why you get it at this level and pass it as a copy?

Reply:
Remember that AppConfig we get from the cache in memory, so it's virtually free to request anywhere.

Reply:
No particular reason. We can move it down inside the function.